src/clients/customerCards/commonTypes.ts

```diff
@@ -0,0 +1,43 @@
+import type { ApiResponse, Options } from '@src/types';
+import type { CustomerCardCardholder } from '../commonTypes';
+
+export declare interface CustomerCardResponse extends ApiResponse {
+  id?: string;
+  customer_id?: string;
+  expiration_month?: number;
+  expiration_year?: number;
+  first_six_digits?: string;
+  last_four_digits?: string;
+  payment_method?: CustomerCardPaymentMethod;
+  security_code?: CustomerCardSecurityCode;
+  issuer?: CustomerCardIssuer;
+  cardholder?: CustomerCardCardholder;
+  date_created?: string;
+  date_last_updated?: string;
+  user_id?: string;
+  live_mode?: boolean;
+}
+
+export declare type CustomerCardPaymentMethod = {
+	id?: string;
+	name?: string;
+	payment_type_id?: string;
+	thumbnail?: string;
+	secure_thumbnail?: string;
+}
+
+export declare type CustomerCardSecurityCode = {
+	length?: number;
+	card_location?: string;
+}
+
+export declare type CustomerCardIssuer = {
+	id?: string;
+	name?: string;
+}
+
+export declare type CustomerCardsGetRemoveData = {
```

Comment:
não deveria estar no singular?
```suggestion
export declare type CustomerCardGetRemoveData = {
```

Reply:
this interface is shared between the get and remove methods of this client